metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -668,7 +668,7 @@ object MetalsEnrichments
   }
 
   implicit class XtensionPositionRange(range: s.Range) {
-    def inString(text: String): String = {
+    def inString(text: String): Option[String] = {
```

Comment:
This function is very cryptic. Only after looking at usages I could guess that input is a textDocument - `range.inString(textDocument.text)` and then what this function does.
Maybe the opposite construction can be more clear - `textDocument.text.atRange(range)`, an extension on `textDocument` - `textAtRange` also makes sense to me.

Reply:
If you agree I can do this refactor in separate PR

Reply:
Sure, it's a bit problematic function anyway :sweat_smile: